versions.props

```diff
@@ -2,7 +2,7 @@ com.fasterxml.jackson.*:* = 2.10.3
 com.github.ben-manes.caffeine:caffeine = 2.8.1
 com.google.code.findbugs:jsr305 = 3.0.2
 com.google.guava:guava = 27.0.1-jre
-com.netflix.concurrency-limits:* = 0.2.0
+com.netflix.concurrency-limits:* = 0.2.2
```

Comment:
Ok only change that looks like it could affect us is the WindowedLimit, but simulations still look ok so :+1:

https://github.com/Netflix/concurrency-limits/compare/v0.2.0...v0.2.2

Reply:
We've been using 0.2.2 for the last 11 months: https://github.com/palantir/conjure-java-runtime/blob/develop/versions.props#L8